server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -0,0 +1,157 @@
+package com.google.musicanalysis.site;
+
+import com.google.gson.*;
+import com.google.musicanalysis.api.musixmatch.*;
+import com.google.musicanalysis.api.naturallanguage.*;
+import com.google.musicanalysis.api.perspective.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/api/analysis")
+public class AnalysisServlet extends HttpServlet {
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse res)
+      throws ServletException, IOException {
+
+    res.getWriter().write("<h1>Song Analysis</h1>\n");
+
+    String trackName = "baby";
+    String artistName = "justin bieber";
+
+    res.getWriter().write("<h3>Lyrics</h3>");
+    String trimmedLyrics = getLyrics(trackName, artistName);
+    res.getWriter().println(trimmedLyrics);
+
+    res.getWriter().write("<h3>Perspective</h3>");
+    HashMap<String, String> perspectiveMap = analyzeWithPerspective(trimmedLyrics);
+    // Print on the screen (won't make it in final cut)
+    Set set = perspectiveMap.entrySet();
+    Iterator mapIterator = set.iterator();
+    while (mapIterator.hasNext()) {
+      Map.Entry entry = (Map.Entry) mapIterator.next();
+      res.getWriter().println(entry.getKey() + ": " + entry.getValue());
+    }
+
+    res.getWriter().write("<h3>NLP</h3>");
+    MagnitudeAndScore nlpObject = analyzeWithNLP(trimmedLyrics);
+    res.getWriter().println("Magnitude: " + nlpObject.magnitude + "Score: " + nlpObject.score);
+  }
+
+  private class MagnitudeAndScore {
+    Double magnitude;
+    Double score;
+
+    public MagnitudeAndScore(Double magnitude, Double score) {
+      this.magnitude = magnitude;
+      this.score = score;
+    }
+  }
+
+  private MagnitudeAndScore analyzeWithNLP(String text) throws IOException {
```

Comment:
could call this `analyzeNLP()`? but no big deal.

Also please add function comments in line 59